cb-talent-development/employer-style-base
# ESB: CB Logo Update

**Purpose**
Per a new press release from Irina, Marketing along with Lorraine Kelly are requesting that we update the logo on the Hiring site and adjust some navigation items in the Software and Services navigation drop down.

This PR also updates the style base so that the navigation changes carry through globally. 

This change impacts the majority of the site and its tablet and mobile variants. When testing, please keep in mind of the following:

- Check the homepage (mobile, tablet, and desktop)
- Check the employment screening page (mobile, tablet, desktop)
- Check any of the Software and Services pages like Talent Discovery or Talent Network (mobile, tablet, and desktop)
- Verify that the mobile flyout menu looks ok
- Verify that breakpoints regarding the subnavigation bars seen in the Software and Services pages look ok
- Verify the logo renders ok in the footer
- Verify All Products sitemap looks ok

**JIRA**
https://careerbuilder.atlassian.net/browse/GCM-11665

**Changes**
* Improvements and fixes
  * New logo with new tagline for the Hiring site
  * Navigation was updated for Software and Services

* Changes to developer setup/environment
  * Employer Style Base version has been updated
  * package.json file was updated to new version of ESB

* Architectural changes
  * Top bar navigation height and variables were adjusted
  * Logo was updated
  * Subnavigation height and offset variables were updated in the style base
  * Footer was adjusted
  * Employment Screening pages were adjusted
  * Software and Services navigation was reorganized
  * All Products sitemap was updated to reflect these new nav items

* Migrations or Steps to Take on Production
  * Mobile styling will need to be verified upon release
  
* Library changes
  * N/A

* Side effects
  * This update will probably affect the position of some items on the hiring site like the top bar navigation and how it reacts in mobile.

**Screenshots**
* Before
![Screen Shot 2020-04-07 at 2 02 14 PM](https://user-images.githubusercontent.com/5348098/78709036-a08bb180-78d8-11ea-989b-1b7830085031.png)

![Screen Shot 2020-04-07 at 2 02 19 PM](https://user-images.githubusercontent.com/5348098/78709039-a1bcde80-78d8-11ea-905a-cf1175417823.png)

![Screen Shot 2020-04-07 at 2 02 28 PM](https://user-images.githubusercontent.com/5348098/78709047-a386a200-78d8-11ea-8c3b-58d9cd0273fa.png)

* After
![Screen Shot 2020-04-07 at 2 03 09 PM](https://user-images.githubusercontent.com/5348098/78709094-b8fbcc00-78d8-11ea-94e8-86757c9c22af.png)

![Screen Shot 2020-04-07 at 2 03 16 PM](https://user-images.githubusercontent.com/5348098/78709101-bac58f80-78d8-11ea-8249-3865ae4e63fe.png)

![Screen Shot 2020-04-07 at 2 03 23 PM](https://user-images.githubusercontent.com/5348098/78709103-bb5e2600-78d8-11ea-97a9-540b6aaee70b.png)

**Feature Server**
http://stg.hiring.careerbuilder.com/

**How to Verify These Changes**
* Specific pages to visit
  * Homepage
  * Post Jobs
  * Employment Screening
  * Resume Database
  * Any of the Software and Services pages (Talent Discovery, Talent Network, Applicant Tracking)
  * The Sitemap

* Steps to take
  * Verify that logo looks ok on the Hiring site in mobile, tablet, and desktop (Marketing is verifying this as well)
  * Verify that floating elements like the subnav seen in the Software and Services pages render out fine
  * Verify that the footer looks ok
  * Verify that items in the Sitemap are correct
  * Verify that the navigation items still work under Software and Services

* Responsive considerations
  * Please verify that the mobile flyout still works as it should with the new logo in place
  * Verify that the additional top bar items in the Employment Screening page render out ok
  * Verify that the floating navigation and subnavigation still appear correctly in Tablet and Mobile views.

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/1221

**Additional Information**
N/A
